app/uk/gov/hmrc/cipphonenumberverification/services/VerifyHelper.scala

```diff
@@ -85,18 +87,24 @@ abstract class VerifyHelper @Inject()(otpService: OtpService, auditService: Audi
   private def sendPasscode(phoneNumberAndOtp: PhoneNumberAndOtp)
                           (implicit hc: HeaderCarrier) = govUkConnector.sendPasscode(phoneNumberAndOtp) map {
     case Left(error) => error.statusCode match {
-      case SERVICE_UNAVAILABLE =>
-        logger.error(error.getMessage())
+      case INTERNAL_SERVER_ERROR =>
+        logger.error(error.getMessage)
         BadGateway(Json.toJson(ErrorResponse(EXTERNAL_SERVICE_FAIL, "Server currently unavailable")))
+      case BAD_REQUEST | FORBIDDEN =>
+        logger.error(error.getMessage)
+        ServiceUnavailable(Json.toJson(ErrorResponse(EXTERNAL_API_FAIL, "External server currently unavailable")))
+      case TOO_MANY_REQUESTS =>
+        logger.error(error.getMessage)
+        TooManyRequests(Json.toJson(ErrorResponse(MESSAGE_THROTTLED_OUT, "The request for the API is throttled as you have exceeded your quota")))
```

Comment:
This is incorrect as I had the following for the two 429 scenarios (when we discussed it as a team during refinement).

THEN  a HTTP response with status code 503 is returned to the client
AND the response body is of type JSON with the properties code and message
AND the code property has the value "EXTERNAL_API_FAIL" 
AND the message property has the value "External server currently unavailable"

The problem of our GovNotify API quotas is not a concern for the client and they do not need to know this.
Therefore if you disagree that the 503, EXTERNAL_API_FAIL then please can we discuss this?
I went for 503 because it is a temporary failure that we can fix and the client should try again soon.
But as said above, please let me know of a suitable alternative, as I don't believe 429 is correct as it is not a client error.
Our quota exceeding is our problem not our clients problem.

Reply:
As discussed at the standup. we will keep the code as is as we should pass on the 429 for rate limits.